docker/witnet-rust/wip0010-recovery.sh

```diff
@@ -0,0 +1,153 @@
+
+# Just a pretty logging helper
+function log {
+  echo "[WIP0010_RECOVERY] $1"
+}
+
+# A helper for calculating ETAs
+function eta {
+  START=$1
+  NOW=$2
+  PROGRESS=$3
+  if [ "$PROGRESS" == "00" ]; then
+    echo "will be shown as synchronization moves forward..."
+  else
+    ELAPSED=$(( NOW - START ))
+    SPEED=$((PROGRESS * 1000 / ELAPSED))
+    REMAINING_PROGRESS=$(( 10000 - PROGRESS ))
+    REMAINING_TIME=$((REMAINING_PROGRESS / (SPEED / 1000) ))
```

Comment:
This line seems to cause some minor problems at the beginning:

```
[WIP0010_RECOVERY] Successfully added healthy peers
[WIP0010_RECOVERY] Still rewinding and synchronizing. Progress: 0.00%. ETA: will be shown as synchronization moves forward...
./wip0010-recovery.sh: línea 64: REMAINING_PROGRESS / (SPEED / 1000) : división por 0 (el elemento de error es "(SPEED / 1000) ")
[WIP0010_RECOVERY] Still rewinding and synchronizing. Progress: . ETA: 
./wip0010-recovery.sh: línea 64: REMAINING_PROGRESS / (SPEED / 1000) : división por 0 (el elemento de error es "(SPEED / 1000) ")
[WIP0010_RECOVERY] Still rewinding and synchronizing. Progress: . ETA: 
```

Reply:
Actually, I think this only happens if the rewind fails and the chain manager dies. Then `nodeStats` returns

    Error: ServerError { code: -32603, message: "MailboxError(Mailbox has closed)" }

Reply:
So this was a consequence of #1870, we can assume that the rewind will not fail.